Resources/public/css/theme/views/loginform.css

```diff
@@ -24,8 +24,9 @@
 }
 
 .ez-view-loginformview .ez-loginform-title {
-    font-size: 0.9rem;
+    font-size: 0.95rem;
     color: #444;
+    font-family: sans-serif;
```

Comment:
It just looked a bit more crips, but could be my screen. I guess we should get @inakijv to have a look at this eventually. Maybe we should use the Maven Pro font here and another places for instance.